sqlglot/dialects/bigquery.py

```diff
@@ -333,3 +333,11 @@ def intersect_op(self, expression: exp.Intersect) -> str:
 
         def with_properties(self, properties: exp.Properties) -> str:
             return self.properties(properties, prefix=self.seg("OPTIONS"))
+
```

Comment:
isn't this copy and pasted? make this a flag on the generator base class and have this done in the base generator